packages/Frontend/src/components/Authenticate.test.jsx

```diff
@@ -17,7 +17,7 @@ describe('Authenticate', () => {
   });
 
   it('should render an error when passed as a prop', () => {
-    assertWithPropsToMatchSnapshot({ error: 'test error' });
+    assertWithPropsToMatchSnapshot({ error: { code: 'errorCode', message: 'test error' } });
```

Comment:
Because errors are now objects with a code and message from Google, rather than a string from Amazon, I had to change this test to match.